web/src/compose_recipient.ts

```diff
@@ -331,6 +331,19 @@ export function initialize(): void {
     $("#private_message_recipient").on("input", restore_placeholder_in_firefox_for_no_input);
 }
 
+export function update_topic_inputbox_on_mandatory_topics_change(): void {
+    if (realm.realm_mandatory_topics) {
+        const $input = $("input#stream_message_recipient_topic");
+        $input.attr("placeholder", "Topic");
```

Comment:
"Topic" should be tagged for translation here, right?

Reply:
Done